### PR TITLE
make regexp match more newline types

### PR DESCRIPTION
**Pull request by Marjoe**
make regexp match more newline types

### CODE REVIEW

File: lib/sig.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ var actionsObjRegexp = new RegExp(
     jsKeyStr + sliceStr   + '|' +
     jsKeyStr + spliceStr  + '|' +
     jsKeyStr + swapStr +
-  '),?\\n?)+)\\};'
+  '),?(\\r\\n|\\r|\\n)?)+)\\};'
```
Comment (fent) [inline lib/sig.js]: Could be simplified as `\\r?\\n?`

Comment (Marjoe) [inline lib/sig.js]: I ran the tests on Windows 10 and failed. Then when I debug the code I found the 'body' in function extractActions in sig.js has all newline as \\r\\n. So I think this is caused by Windows.

Comment (fent) [inline lib/sig.js]: Yes but, wouldn't `\\r?\\n?` also match that string? Could you test that on Windows?

Comment (Marjoe) [inline lib/sig.js]: `\\r?\\n?` works well. Thanks for your guide:)

**Comment by fent** [comment]
Thank you!

**Pull request merged**